nipype/interfaces/ants/registration.py

```diff
@@ -379,6 +379,8 @@ class RegistrationInputSpec(ANTSCommandInputSpec):
         low=0.0, high=1.0, value=0.0, argstr='%s', usedefault=True, desc="The Lower quantile to clip image ranges")
 
     verbose = traits.Bool(argstr='-v', default=False)
+    profiling = traits.Bool(False, usedefault=True,
+                            desc='generate profiling output fields')
```

Comment:
Not really.

My original idea is that we grow the number of outputs that are parsed from the log. When there are a bunch, there's a good chance that logs formatting change or we make mistakes. Being an added feature (and nonfunctional), it could come handy to have a switch to avoid parsing the stdout.

But I have no concern with removing the input.